autoload/fzf/vim.vim

```diff
@@ -576,8 +576,9 @@ function! s:bufopen(lines)
 endfunction
 
 function! s:format_buffer(b)
+  let filename_modifier = get(g:, 'fzf_buffer_fnamemodify_arg', ":~:.")
```

Comment:
You can set the `g:fzf_buffer_fnamemodifier_arg` variable in your .vimrc. Otherwise, it will default to the existing filename_modifier: ":~:."